tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "types": ["node", "jest"],
     "esModuleInterop": true,
     "noImplicitOverride": true,
+    "skipLibCheck": true,
```

Comment:
what's this do ?

Reply:
https://www.typescriptlang.org/tsconfig#skipLibCheck